integration/helpers.go

```diff
@@ -1265,11 +1265,11 @@ func (s *discardServer) Stop() {
 	s.sshServer.Close()
 }
 
-func (s *discardServer) HandleNewChan(conn net.Conn, sconn *ssh.ServerConn, newChannel ssh.NewChannel) {
+func (s *discardServer) HandleNewChan(ccx *sshutils.ConnectionContext, newChannel ssh.NewChannel) {
```

Comment:
nit: `ctx`?

Reply:
I deliberately assumed the convention of referring to `sshutils.ConnectionContext` as `ccx` to handle the fact that it is frequently in-scope at the same time as `srv.ServerContext`, which is already conventionally called `ctx`.